### DISCUSSION TITLE
What is SASL mechanism to be given while trying to connect to MSK using AWS_MSK_IAM mode

### DISCUSSION THREAD

**Discussion by samanton6** [aws/aws-msk-iam-sasl-signer-go#18] [Q&A]
What SASL mechanism to be given while trying to connect to MSK enabled as IAM because using config.Net.SASL.Mechanism = sarama.SASLTypeOAuth we are getting error saying "Broker does not support required SASL mechanism"

**Comment by sankalpbhatia** [comment]
Is this an MSK version which is lower than 2.7.1? Note that this feature is only available for versions 2.7.1 and above

**Reply by samanton6** [reply]
Yes, our MSK version is on 2.7.0, We will upgrade to 2.7.1 and will again validate
